prow/config.yaml

```diff
@@ -40,6 +40,17 @@ deck:
     announcement: "This page -- spyglass -- is the default viewer for prow.{{if .ArtifactPath}} For now, the deprecated gubernator page spyglass replaces <a href='https://gubernator.k8s.io/build/{{.ArtifactPath}}'>remains available</a>.{{end}}"
 
   tide_update_period: 1s
+  # Basing this on istio/test-infra/OWNERS for now.
+  rerun_auth_config:
+    authorized_users:
```

Comment:
This is outdated after https://github.com/kubernetes/test-infra/pull/13564
`authorized_users` will need to become `github_users` after that PR is applied.